getting-started/generative-ai.rst

```diff
@@ -24,3 +24,11 @@ Unacceptable uses
 Maintainers may close issues and PRs that are not useful or productive, including
 those that are fully generated by AI. If a contributor repeatedly opens unproductive
 issues or PRs, they may be blocked.
+
+Anti-patterns
+=============
+- While AI-assisted tools such as autocompletion can enhance productivity, they sometimes rewrite entire code blocks instead of making small, focused edits.
```

Comment:
I'd actually tighten this phrasing up a bit.  I find this list of anti-patterns to not be AI specific.  We already reject PRs coming from people who make larger than necessary changes that are difficult to review.  So I'd suggest wording it as a "positive desired thing, not negative example" phrasing.

- Make small focused edits, prefer not rewriting entire code blocks.
   This makes it easier to review changes and fully understand both the original intent of the code and the rationale behind the new modifications.
   ...